packages/app/src/providers/h5wasm/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { EntityKind } from '@h5web/shared';
+
+import type {
+  H5WasmDatasetReponse,
+  H5WasmEntityResponse,
+  H5WasmExternalLinkResponse,
+  H5WasmGroupResponse,
+  H5WasmSoftLinkResponse,
+} from './models';
+
+export function isGroupResponse(
+  response: H5WasmEntityResponse
+): response is H5WasmGroupResponse {
+  return response.type === EntityKind.Group;
+}
```

Comment:
I'm happy to change the constants in h5wasm to be lower case, though I'm not sure why e.g. EntityKind.Group is not "Group" since they refer to class or interface names.

Reply:
I think this can be traced back to HSDS. We also use lower-case in H5Grove. At this point it's just a convention, I guess. No need to change it in h5wasm, though -- a simple `response.type.toLowerCase()` here would suffice.